modules/claims_api/spec/factories/claims_api_base_factory.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  sequence(:uuid, &:to_s)
+  sequence(:factory_id) { |n| (n + 1).to_s }
+
+  factory :claims_api_base_factory, class: Hash do
+    id { FactoryBot.generate(:uuid) }
+    status { %w[pending errored submitted].sample }
+    auth_headers { { test: ('a'..'z').to_a.shuffle.join } }
+    cid {
+      %w[0oa9uf05lgXYk6ZXn297 0oa66qzxiq37neilh297 0oadnb0o063rsPupH297 0oadnb1x4blVaQ5iY297
+         0oadnavva9u5F6vRz297 0oagdm49ygCSJTp8X297 0oaqzbqj9wGOCJBG8297 0oao7p92peuKEvQ73297].sample
+    }
+    created_at { Time.zone.now }
+    updated_at { Time.zone.now }
+  end
+
+  trait :with_full_headers do
```

Comment:
Why the name change here?

Reply:
@rockwellwindsor-va Yeah, good question! There are more tests that reference :with_full_headers, than :with_full_headers_jesse and I don't think we need two attributes that are exactly the same, just with different names.